src/main/java/AddCommand.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * A command to add a task to Duke.
+ */
+public class AddCommand extends Command {
+    private Task task;
+
+    public AddCommand(Task t) {
+        this.task = t;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws SavingException {

```

Comment:
Interesting use of SavingException as a subclass of DukeException, makes the code more readable good job